app/shared/resource-map/mapSelector.js

```diff
@@ -19,7 +19,7 @@ const shouldMapFitBoundariesSelector = createSelector(
   urlSearchFiltersSelector,
   selectedUnitIdSelector,
   (filters, unitId) => (
-    Boolean(find(omit(filters, ['date', 'duration', 'end', 'page', 'start']), filter => filter !== '' && filter !== false)) || Boolean(unitId)
+    Boolean(find(omit(filters, ['date', 'duration', 'end', 'page', 'start', 'municipality']), filter => filter !== '' && filter !== false)) || Boolean(unitId)
```

Comment:
This line might look better if it gets broken in two

Reply:
Well linter didnt complain so im just leave it be :D :D